packages/react/src/alertbox/Alertbox.tsx

```diff
@@ -0,0 +1,200 @@
+import { Children } from 'react';
+import { cva, type VariantProps, cx } from 'cva';
+import { useLocale, Button } from 'react-aria-components';
+import {
+  Close,
+  InfoCircle,
+  CheckCircle,
+  Warning,
+  CloseCircle,
+} from '@obosbbl/grunnmuren-icons-react';
+import { useState } from 'react';
+
+// TODO: expand/collapse
+// TODO: add new icons
+
+const iconMap = {
+  info: InfoCircle,
+  success: CheckCircle,
+  warning: Warning,
+  danger: CloseCircle,
+};
+
+const alertVariants = cva({
+  base: [
+    'grid grid-cols-[auto_1fr_auto] items-center gap-x-2 gap-y-4 rounded-md border-2 px-3 py-2',
+  ],
+  variants: {
+    /**
+     * The variant of the alert
+     * @default info
+     */
+    variant: {
+      info: 'border-[#1A7FA7] bg-sky-light',
+      success: 'border-[#0F9B6E] bg-mint-light',
+      warning: 'border-[#C57C13] bg-[#FFF2DE]',
+      danger: 'border-[#C0385D] bg-red-light',
+    },
+  },
+  defaultVariants: {
+    variant: 'info',
+  },
+});
+
+type Props = VariantProps<typeof alertVariants> & {
+  children: React.ReactNode;
+  /**
+   * The ARIA role for the alertbox.
+   */
+  role: 'alert' | 'status' | 'dialog' | 'presentation' | 'none';
```

Comment:
Burde vi dra denne ned til kun `alert`,  `status` og `presentation` for å gjøre bruken litt mindre forvirrende? Når jeg så dette måtte jeg selv sjekke om det egentlig er noen forskjell på `presentation` og `none` (det er det ikke), så tenker at her burde vi gjøre det vi kan for å gjøre det så enkelt som mulig for konsumenten av komponten.

`dialog` ser jeg ikke som realistisk å bruke på denne? For da er jo det noe vi må bake inn i komponenten på noe vis, type toast etc.

Reply:
Ja, enig i at det kan være forvirrende. Grunnen til at jeg tok med begge var fordi jeg tenkte at noen kanskje har et forhold til `presentation`, og er vant til å bruke det. For de som ikke er kjent med det vil jeg tro at `none` er relativt åpenbart. Men er som du sier, ingen forskjell på de to. Jeg stemmer i så fall for å beholde `none`, siden det er mest beskrivende. Hva tenker du?

Enig i at `dialog` er litt på kanten, grunnen til at jeg tok det med var at jeg så for meg at folk kanskje blir litt kreative i hvordan de bruker den (toast som du sier). Men også enig i at det kanskje er dumt å indikere at det er tiltenkt bruk, før vi har mer toast-funksjonalitet på plass.

Mitt forslag er å endre til:

`role: 'alert' | 'status' | 'none';`